parse_test.go

```diff
@@ -0,0 +1,81 @@
+package goconf
+
+import (
+	"github.com/stretchr/testify/assert"
```

Comment:
we need to format the imports. use the makefile formatter. `make format` should do the trick :)